docs/user/FlowVariables.md

```diff
@@ -0,0 +1,268 @@
+﻿# Environment Variables for the OpenROAD Flow Scripts
+
+
+Environment variables are used in the OpenROAD flow to define various
+platform, design and tool specific variables to allow finer control and
+user overrides at various flow stages. These are defined in the
+`config.mk` file located in the platform and design specific directories.
+
+
+## Platform
+
+
+These variables must be defined in the platform specific config file
+located in the OpenROAD-flow-scripts directory of
+`./flow/platforms/<PLATFORM>/config.mk`
+
+
+Note: The variable `PLATFORM_DIR` is set by default based on the `PLATFORM`
+variable. For OpenROAD Flow Scripts we have the following public platforms:
+
+
+-   `sky130hd`
+-   `sky130hs`
+-   `nangate45`
+-   `asap7`
+
+
+## Platform Specific Environment Variables
+
+
+The table below lists the complete set of variables used in each of the
+public platforms supported by the OpenROAD flow.
+
+
+Note:
+-   = indicates default definition assigned by the tool
+-   ?= indicates that the variable value may be reassigned
+-   N/A indicates that the variable/files is not supported currently.
+
+
+| **Configuration Variable**           | **sky130hd** | **sky130hs** | **nangate45** | **asap7** |
+|--------------------------------------|--------------|--------------|---------------|-----------|
+| Library Setup                        |              |              |               |           |
+| `PROCESS`                            | =            | =            | =             | =         |
+| `TECH_LEF`                           | =            | =            | =             | =         |
+| `SC_LEF`                             | =            | =            | =             | =         |
+| `LIB_FILES`                          | =            | =            | =             | =         |
+| `GDS_FILES`                          | =            | =            | =             | =         |
+| `DONT_USE_CELLS`                     | =            | =            | =             | =         |
+| Synthesis                            |              |              |               |           |
+| `LATCH_MAP_FILE`                     | =            | =            | =             | =         |
+| `CLKGATE_MAP_FILE`                   | =            | =            | =             | =         |
+| `ADDER_MAP_FILE`                     | ?=           | ?=           | ?=            | ?=        |
+| `BLACKBOX_MAP_TCL`                   | N/A          | N/A          | N/A           | =         |
+| `TIEHI_CELL_AND_PORT`                | =            | =            | =             | =         |
+| `TIELO_CELL_AND_PORT`                | =            | =            | =             | =         |
+| `MIN_BUF_CELL_AND_PORTS`             | =            | =            | =             | =         |
+| `ABC_CLOCK_PERIOD_IN_PS`             | N/A          | =            | ?=            | ?=        |

```

Comment:
Why are sky* inconsistent here?

Reply:
@maliberty 
sky130 platform config.mk have ABC_CLOCK_PERIOD_IN_PS but someone commented and made PR. 
This variable list created based on sky130/nangate45/asap7 platform.
Then it should be clean-up from platform config.mk.